xapian-core/tests/harness/backendmanager.cc

```diff
@@ -264,6 +264,11 @@ BackendManager::clean_up()
 {
 }
 
+void
+BackendManager::killall_server()
+{
+}
```

Comment:
If there's no implementation for the current backend, doing nothing isn't a good option - better to throw an exception so we quickly see if testcases requiring this feature are run with backends which we don't support it for.